_posts/en/newsletters/2020-09-23-newsletter.md

```diff
@@ -0,0 +1,264 @@
+  protocols.  This has lead to partial solutions such as [anchor
+  outputs][topic anchor outputs] that work with specific protocols (e.g.
+  two-party LN channels) but aren't easy to generalize and so are
+  somewhat unsatisfactory.
+
+    At the consensus layer, Rubin proposes allowing transactions to
+    optionally contain a special final output that commits to the txids
+    of one or more other unconfirmed transactions.  The transaction with
+    the special output, a *sponsor* transaction, may only be included in
+    a valid block if every one of the transactions it *sponsored* is
+    also included in that same block.  This means a miner who wants the
+    high feerate from a sponsor transaction will be incentivized to also
+    confirm the sponsored transactions even if those transactions pay a
+    low feerate.  Beyond their special sponsorship output, sponsor
+    transactions are normal Bitcoin transactions.
+
+    At the policy layer that controls mempool acceptance and transaction
+    relay, Rubin suggests a set of simple rules that allow anyone to
+    sponsor any transaction currently in a mempool or to replace an
+    existing sponsor transaction with higher feerate alternative that
+    makes the same commitment.  The goal is to ensure that, as long as a
+    sponsor transaction follows the rules and pays a high enough
+    feerate, it will propagate across the network without running afoul
+    of pinning or other attacks.
+
+    Rubin's proposal comes with a [reference implementation][rubin
+    refimpl] and discussion of design tradeoffs and forward
+    compatibility.  As of this writing, comments on the list have been
+    positive but non-committal---the proposal needs to not only provide
+    satisfactory solutions to existing problems but prove that it is
+    both absent of any significant new problems and that it will attract
+    the support needed to introduce a change to Bitcoin's consensus
+    rules.
+
+- **Research into conflicts between timelocks and heightlocks:** A
+  [post][blockstream post] to the Blockstream engineering blog describes
+  an interaction between different types of Bitcoin time locks
+  (timelocks) and block-height locks (heightlocks).  All transactions
+  since Bitcoin's initial release have had an `nLockTime` field.  A soft
+  fork activating at block 31,000 (December 2009) began
+  [comparing][time-height-lock fork] this field against a block header's
+  explicit nTime field and its implicit height (number of blocks since
+  the Genesis Block).  Using that comparison, a block is only valid if
+  every one of its transactions follows these two
+  rules:[^final-sequence]
+
+    - If a transaction's nLockTime is below 500 million, it must also be
+      below the block's height.
+
+    - If a transaction's nLockTime is equal to or above 500 million, it
+      must be below the block header's nTime (in [epoch time][]).
+
+    This overloading of the single nLockTime field for two different
```

Comment:
Unsure if we want to use `nLockTime` or nLockTime after the first use of the term; same below in lines 88, 89, and 104, and in the Footnotes.

Reply:
When repeatedly referring to something over an extended description, I think it's nicer to read if we don't keep using special formatting.  For shorter texts, like our typical one-paragraph merge descriptions, I think it's better to be consistent by reusing code formatting for configuration parameters, RPC names, etc...

Thanks for being on the lookout for inconsistencies, though.  :-)

Reply:
Yes, I agree it's nicer to read without code formatting all over.